Applications/Spire/Source/Ui/SideBox.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Spire/Ui/SideBox.hpp"
+#include "Spire/Spire/LocalValueModel.hpp"
+#include "Spire/Ui/ArrayListModel.hpp"
+
+using namespace Nexus;
+using namespace Spire;
+
+SideBox* Spire::make_side_box(QWidget* parent) {
+  return make_side_box(Side::NONE, parent);
```

Comment:
```Side::NONE``` is not a valid case, the components.xd specifies that only Buy and Sell are valid.